### ISSUE TITLE
I can't create an encrypted volume on the SD card

### ISSUE THREAD

**Issue by Olejja** [hardcore-sushi/DroidFS#233]
Hello.

I can't create an encrypted volume on the SD card.
I click add (green folder), after which android asks for a pattern, select sdcard1 in Explorer and press select. And it takes me back to droidfs to the place where I clicked on the green folder. The path remains empty.

On the README page in the limitations section that:

```
Due to Android's storage restrictions, encrypted volumes located on SD cards must be placed under /Android/data/sushi.hardcore.droidfs/ if you want DroidFS to be able to modify them.
```

I tried to register this path, but a warning popped up that droidfs does not have permission to write to this directory (I don’t even know if it exists).

Does this mean I need to root my device? I wouldn't want to do that, to be honest! By the way, I'm using Android 7.1.2. What should I do to create and use encrypted volumes on the SD card?

**Comment by hardcore-sushi** [comment]
Which DroidFS version are you using?

Try to create the `/Android/data/sushi.hardcore.droidfs/` directory before selecting it.

Also, please attach your logcat recorded when you select a folder. It will give more information.

**Comment by Olejja** [comment]
>Which DroidFS version are you using?

Oh sorry. DroidFS version 2.0.2 from F-Droid.

>Try to create the /Android/data/sushi.hardcore.droidfs/ directory before selecting it.

I already tried it, the same window pops up. I also tried using `/sdcard1/Android/data/sushi.hardcore.droidfs/`

**Comment by Olejja** [comment]
I looked through Explorer for the path to the file on the card and used it as a pointer. It turned out like this:

`/storage/4357-B0AC/Android/data/sushi.hardcore.droidfs/`

It gave me the error "The encrypted volume is not recognized. Check the selected path."

**Comment by Olejja** [comment]
I've made dozens of attempts to create an encrypted volume on the SD card, but with no success. Rebooted the smartphone. I made several more attempts, but also failed.

Then everything suddenly worked! All this time I was creating the desired directory on the SD card through the file manager and using it I copied the path to it. As a result, the path is exactly the same as it was, that is `/storage/4357-B0AC/Android/data/sushi.hardcore.droidfs/`. I don't know what it was.

The topic can be considered resolved, but I have one more question: **Do I understand correctly that I will not be able to work with volumes through a computer?**
